service/src/main/java/com/ge/predix/acs/privilege/management/ZoneDoesNotExistException.java

```diff
@@ -0,0 +1,44 @@
+package com.ge.predix.acs.privilege.management;
+
+public class ZoneDoesNotExistException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    public ZoneDoesNotExistException() {
+        super();
+    }
+
+    /**
+     * @param message
+     * @param cause
+     * @param enableSuppression
+     * @param writableStackTrace
+     */
+    public ZoneDoesNotExistException(final String message, final Throwable cause, final boolean enableSuppression,
+                                     final boolean writableStackTrace) {
+        super(message, cause, enableSuppression, writableStackTrace);
+    }
+
+    /**
+     * @param message
+     * @param cause
+     */
+    public ZoneDoesNotExistException(final String message, final Throwable cause) {
+        super(message, cause);
+    }
+
+    /**
+     * @param message
+     */
```

Comment:
These JavaDocs don't add any new information that the method signature doesn't provide.